src/cargo/core/source/source_id.rs

```diff
@@ -39,6 +39,10 @@ struct SourceIdInner {
     /// WARNING: this is not always set for alt-registries when the name is
     /// not known.
     name: Option<String>,
+    /// Name of the alt registry in the [registries] table.
```

Comment:
Looks like the docs job failed. I think this needs to be escaped so it isn't interpreted as a markdown link.

```suggestion
    /// Name of the alt registry in the `[registries]` table.
```